### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    [resources.system_user]
+    allow_ssh = true
+
+    [resources.install_dir]
+
+    [resources.data_dir]
+
+    [resources.permissions]
+    main.url = "/"
+    admin.allowed = ["admin"]
```
Comment (alexAubin) [inline manifest.toml:58]:
```suggestion
    admin.allowed = ["admins"]
```

File: manifest.toml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+[install]
+    [install.domain]
+    type = "domain"
+
+    [install.path]
+    type = "path"
+    default = "/gitea"
+
+    [install.admin]
+    type = "user"
```
Comment (alexAubin) [inline manifest.toml:38]: I'm not sure this question is still useful now if we go with `admin.allowed = ["admins"]` below ...

Or another way is to have a question named `init_admin_permission` and remove the `admin.allowed` line (but then we still need something to declare that the admin permission does exists

File: scripts/upgrade [hunk 3]
```diff
@@ -3,310 +3,91 @@
...
-    restart_gitea
-;&
-"1.16."* )
-    ynh_setup_source $final_path source/${architecture}_1.16
-    restart_gitea
-;&
-"1.16."* )
-    ynh_setup_source $final_path source/${architecture}_1.17
-    restart_gitea
-;&
```
Comment (Josue-T) [inline scripts/upgrade:243]: As already explained many time like [here](https://github.com/YunoHost-Apps/gitea_ynh/pull/45#discussion_r884315843), please keep this as it is.

**Review by Josue-T — CHANGES_REQUESTED** [review]

**Comment by ericgaspar** [comment]
!testme

**Pull request closed without merging**